src/_guides/language/language-tour.md

```diff
@@ -610,8 +610,8 @@ You can create a “raw” string by prefixing it with `r`:
 var s = r'In a raw string, not even \n gets special treatment.';
 ```
 
-See [Runes](#runes) for details on how to express Unicode
-characters in a string.
+See [Runes and grapheme clusters](#runes) for details on how
```

Comment:
I suggest we make the anchor link `#characters` given that's the recommendation

Reply:
I was trying to preserve the existing anchor name. Probably not a big issue, though.